pyaerocom/aeroval/setupclasses.py

```diff
@@ -538,10 +539,9 @@ def to_json(self, outdir: str, ignore_nan: bool = True, indent: int = 3) -> None
             json indentation
 
         """
-        filepath = os.path.join(outdir, self.json_filename)
-        data = self.json_repr()
-        write_json(data, filepath, ignore_nan=ignore_nan, indent=indent)
-        return filepath
+        with aerovaldb.open(f"json_files:{outdir}") as db:
+            with db.lock():
```

Comment:
`json_files:{outdir}` must be a configuration option. The user needs to be able to switch to another implementation of aerovaldb from the configuration.